springfield/cms/templates/components/uitour_button.html

```diff
@@ -0,0 +1,27 @@
+{#
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<div class="ui-tour">
+  <button class="button {{ theme_class }}" {% if disabled %}disabled{% endif %}>
```

Comment:
Let's add a `type="button"` on here just to be safe.
```suggestion
  <button class="button {{ theme_class }}" type="button" {% if disabled %}disabled{% endif %}>
```